src/components/data-pane/field-list.tsx

```diff
@@ -1,49 +1,57 @@
 import * as React from 'react';
 import * as CSSModules from 'react-css-modules';
 import {connect} from 'react-redux';
+import * as TetherComponent from 'react-tether';
 import * as styles from './field-list.scss';
 
+import {ExpandedType} from 'compassql/build/src/query/expandedtype';
+import {PrimitiveType, Schema} from 'compassql/build/src/schema';
+import {ShelfFieldDef} from '../../../build/src/models/shelf/encoding';
```

Comment:
This should be `../../models/shelf/encoding` (we should import from original source, not the compiled ones.)  

I'll add 
```
"tsimporter.filesToExclude": ["build/**"],
```

to `master`